client/galaxy/scripts/mvc/form/form-input.js

```diff
@@ -124,6 +124,9 @@ define([], function() {
             tmp +=              '<div class="ui-table-form-info">';
             if (options.help) {
                 tmp +=              options.help;
+                if (options.argument) {
```

Comment:
Maybe here we should check that `options.help` does not already contain `options.argument` to make it possible for tool developers to cleanly support old and new Galaxy syntax.